src/Voyager.php

```diff
@@ -353,4 +353,9 @@ public function getLocales()
 
         return $allLocales;
     }
+
+    public function getSeedsFolderName()
```

Comment:
Not too sure I like having this exposed through the Voyager facade.  Doesn't feel like the right place (though I don't really have a better solution at the moment).  Facade just feels like it should be more of an external API, whereas stuff like this is 100% internal (and honestly not even Voyager-specific)

Reply:
Yes, I agree. That was the easier place but probably not the best suited.
I could create a new class with an handle method or put it in an helper :confused: 
What do you think?

Reply:
Helper functions are also exposed globally, so they pollute the app.  Not really a fan personally.  Let's go with a "helper class".  A class with static methods that can be used within Voyager, but don't necessarily need to be documented anywhere

Reply:
Done, not sure about class name and position but I could not put it in Helpers and not in Actions.